### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by recursiveribbons**
- Added Dockerfile and docker-compose.yml template
- Enable bot to read environment variables in addition to the config.ini file. Env variables have priority.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM python:3
+WORKDIR /usr/src/app
+RUN pip3 install --user python-telegram-bot dataset cachetools requests Jinja2 mysqlclient
```
Comment (TheLastProject) [inline Dockerfile]: Please use the requirements.txt file now that it exists :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+COPY . .
```
Comment (TheLastProject) [inline Dockerfile]: I'd like this Dockerfile to provide a volume so one could easily mount the configuration and wouldn't have to rebuild the container just to change the configuration.

Comment (recursiveribbons) [reply Dockerfile]: The config can be changed using the env vars in docker-compose. They have higher prio than the config file.

Comment (TheLastProject) [reply Dockerfile]: Good enough for now I guess, yeah

**Pull request merged**